### PR TITLE
Feature/328 mobile offer list

### PR DESCRIPTION
**Pull request by dylanalvarez**
image
image
image
image

### CODE REVIEW

File: src/pages/Applicant/Home/Feed/Filters/component.tsx [hunk 1]
```diff
@@ -13,21 +17,55 @@ export const Filters: FunctionComponent<IFiltersProps> = (
...
+      <div className={styles.filters}>
+        <div className={styles.titleContainer}>
+          <Headline className={styles.title}>{translations?.title}</Headline>
+          <div>
+            <Button
+              className={styles.openMobileApplyFilters}
+              kind="primary"
+              onClick={() => setFiltersAreOpen(true)}
+            >
+              Filtrar
```
Comment (BlancoSebastianEzequiel) [inline src/pages/Applicant/Home/Feed/Filters/component.tsx:38]: Agregaria la translation

Comment (dylanalvarez) [reply src/pages/Applicant/Home/Feed/Filters/component.tsx:38]: https://github.com/fiuba-laboral-v2/back-end/pull/246

File: src/pages/Applicant/Home/Feed/Filters/component.tsx [hunk 2]
```diff
@@ -13,21 +17,55 @@ export const Filters: FunctionComponent<IFiltersProps> = (
...
+          className={styles.appliedFilters}
+          filter={filter}
+          careers={careers}
+        />
+        <ApplyFilters
+          withTitle
+          className={styles.desktopApplyFilters}
+          {...applyFiltersProps}
+        />
+        <Dialog fullScreen open={filtersAreOpen}>
```
Comment (BlancoSebastianEzequiel) [inline src/pages/Applicant/Home/Feed/Filters/component.tsx:52]: Por que un Dialog?

File: src/pages/Applicant/Home/Feed/Filters/component.tsx [hunk 3]
```diff
@@ -13,21 +17,55 @@ export const Filters: FunctionComponent<IFiltersProps> = (
...
-        careers={careers}
-      />
-      <ApplyFilters
-        className={styles.applyFilters}
-        filter={filter}
-        careers={careers}
-        translations={translations}
-      />
+) => {
+  const [filtersAreOpen, setFiltersAreOpen] = useState(false);
```
Comment (BlancoSebastianEzequiel) [inline src/pages/Applicant/Home/Feed/Filters/component.tsx:21]: filtersAreApplied

Comment (llmanuel) [reply src/pages/Applicant/Home/Feed/Filters/component.tsx:21]: +1

**Pull request merged**